### ISSUE TITLE
Logger false would return "Cannot read property 'getTimestamp' of undefined"

### ISSUE THREAD

**Issue by adlanarifzr** [nestjs/nest#5959]
## Current behavior

If logger is set in the callback of the `app.listen` when starting the app, it would return the following error.
```ts
TypeError: Cannot read property 'getTimestamp' of undefined
    at Function.printMessage (C:\docker\nest-app\node_modules\@nestjs\common\services\logger.service.js:99:55)
    at Function.verbose (C:\docker\nest-app\node_modules\@nestjs\common\services\logger.service.js:62:14)
    at Server.<anonymous> (C:\docker\nest-app\dist\apps\api\main.js:17:25)
    at Object.onceWrapper (events.js:288:20)
    at Server.emit (events.js:200:13)
    at emitListeningNT (net.js:1262:10)
    at processTicksAndRejections (internal/process/task_queues.js:74:11)
```

## Input Code

```ts
async function bootstrap() {
	// Create the application
	const app = await NestFactory.create(AppModule, {
		logger: false,
	});

	// Start the server
	await app.listen(
		3000,
		'127.0.0.1',
		() => {
			Logger.verbose(`Server is running on https://127.0.0.1:3000`, 'Server');
		}
	);
}
bootstrap();
```

## Expected behavior

It should just run while ignoring the logger.

## Environment

<pre><code>
Nest version: 7.6.1

**Comment by kamilmysliwiec** [comment]
Fixed in 7.6.2. Thanks for reporting

**Comment by Erchoc** [comment]
#6175 maybe not fixed.
